Taskfile.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+tasks:
```

Comment:
Nitpick : je pense que tu peux créer tout de suite un namespace pour docker.

Et on aurait :
'task docker build'
'task docker run'
Dans le future y aura vraisemblablement un publish.

'''
version: '3'

tasks:
  docker:
    desc: Namespace for Docker tasks
    aliases: [d]
    cmds:
      - task: docker:build

  docker:build:
    desc: Build Docker image
    cmds:
      - docker build -t my-image .
'''